### ISSUE TITLE
Table formula is changed on re-saving

### ISSUE THREAD

**Issue by leokr** [EPPlusSoftware/EPPlus#1234]
Just re-saving my file with EPPlus cause it to change formula.
Repro on 7.0.5, not repro on 6.2.15
Probably that case is very similar to #95

```
xlPackage = New ExcelPackage(New IO.FileInfo(SaveFileDialog1.FileName))
xlPackage.SaveAs(New IO.FileInfo(SaveFileDialog1.FileName))
```

In my original formula "-" sign placed on new line, after line break (green on screenshot)
After re-saving file with EPPlus "-" goes to previous line, before line break (yellow on screenshot).
That also triggers excel to show "inconsistent formula" warning

![image](https://github.com/EPPlusSoftware/EPPlus/assets/37657587/a0223a5b-5213-4c18-81d2-94a01ef16dca)

 here is original and resaved versions of excel file with issue:
[EppRepro1234.orig.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/13818599/EppRepro1234.orig.xlsx)
[EppRepro1234.resaved.8fc24c7de039.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/13818607/EppRepro1234.resaved.8fc24c7de039.xlsx)

**Comment by swmal** [comment]
Thanks for reporting, we'll have a look at this.

**Comment by leokr** [comment]
thanks, I attached .xlsx files to my first post

**Comment by JanKallman** [comment]
I have added a fix for this issue. The problem was that the tokenizer switched the whitespace and the negator token when updating the formula namespaces, causing the whitespaces to appear after the negator.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.0.6
